TYPO3.TYPO3CR/Classes/TYPO3/TYPO3CR/Domain/Model/Workspace.php

```diff
@@ -560,7 +565,7 @@ protected function findNodeDataInTargetWorkspace(NodeInterface $node, Workspace
      * @return void
      * @Flow\Signal
      */
-    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace, Workspace $newBaseWorkspace)
+    protected function emitBaseWorkspaceChanged(Workspace $workspace, Workspace $oldBaseWorkspace = null, Workspace $newBaseWorkspace = null)
```

Comment:
Forgot to mention it: yes, it is related.

With the nested workspace in Neos 2.1, you can theoretically create a workspace without a base workspace (that means on the base level, like "live"). And also theoretically you can rebase that workspace later on. In that case `$oldBaseWorkspace` would be `null`. 

I stumbled over it when I wanted to set the base workspace in the unit test of this change.

Question is if that is considered to be breaking – the signal was only introduced in 2.1 and I really doubt that anyone actually discovered it or is using it already ;-) But on the other hand, you never know of course.